inflammation/models.py

```diff
@@ -41,3 +41,13 @@ def daily_min(data):
     :return: Numpy array with the minimum daily value."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - mmm) * (entry - mmm))
+
+    s_dev2 = sum(devs) / len(data)
```

Comment:
heuy, you could use the function np.std instead of calculating the standard edviation from scratch :)